src/globalStyle.css

```diff
@@ -1,25 +1,33 @@
-:root{
+:root {
   --grey: rgb(107, 107, 107);
-  --correct: #3BCB67;
-  --absent: #FF6671;
-  --misplaced: #FFDD82;
+  --correct: #3bcb67;
+  --absent: #ff6671;
+  --misplaced: #ffdd82;
   --white: #f5f5f5;
   --black: rgb(32, 32, 32);
   --bg: var(--white);
   --bg-alt1: #bbbbbb;
   --bg-alt2: rgb(207, 207, 207);
   --input-color: var(--white);
   --shadow: rgba(51, 51, 51, 0.6);
-  
+
   font-size: clamp(16px, calc(0.5rem + 0.6vw), 1.5rem);
   --size-xs: 0.1rem;
   --size-s: 0.3rem;
   --size-n: 1rem;
   --size-l: 1.2rem;
   --size-xl: 1.5rem;
+
+  --tile-gap: 0.5rem;
```

Comment:
Instead of using margins on the tiles. I set a gap variable and use the more semantic `gap` property on the grids.